webapp/src/components/FormattedBarcode.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <span v-if="barcode">
+    <font-awesome-icon
+      v-if="enableBarcode"
+      icon="barcode"
+      class="badge clickable"
+      title="Click to show Barcode for this item"
+      aria-label="Click to show Barcode for this item"
+      @click="BarcodeModalOpen = true"
+    />
+    <font-awesome-icon v-else icon="barcode" class="badge" />
+    <span class="badge formatted-barcode" :style="{ backgroundColor: badgeColor }">
```

Comment:
```suggestion
    <span class="badge" :style="{ backgroundColor: badgeColor }">
```

Not used. Is it intended that the barcode icon and the span are not vertically aligned?